logfire/_internal/integrations/psycopg.py

```diff
@@ -11,13 +11,25 @@
 if TYPE_CHECKING:  # pragma: no cover
     from opentelemetry.instrumentation.psycopg import PsycopgInstrumentor
     from opentelemetry.instrumentation.psycopg2 import Psycopg2Instrumentor
+    from typing_extensions import TypedDict, Unpack
 
     Instrumentor = PsycopgInstrumentor | Psycopg2Instrumentor
 
+    class CommenterOptions(TypedDict, total=False):
+        db_driver: bool
+        db_framework: bool
+        opentelemetry_values: bool
+
+    class PsycopgInstrumentKwargs(TypedDict, total=False):
+        enable_commenter: bool
+        commenter_options: CommenterOptions
+        skip_dep_check: bool
```

Comment:
We already pass this option.

Reply:
Ah, you mean that it will be ignored on the user's side.

I'll remove it.